light/proxy/routes.go

```diff
@@ -133,6 +134,22 @@ func makeCommitFunc(c *lrpc.Client) rpcCommitFunc {
 	}
 }
 
+type rpcDataCommitmentFunc func(
+	ctx *rpctypes.Context,
+	beginBlock uint64,
+	endBlock uint64,
```

Comment:
[minor]
For consistency with `rpcBlockchainInfoFunc` in line 80 above (and thus readability), I would:
* use `int64`
* rename `beginBlock` and `endBlock` to `minHeight` and `maxHeight`

Reply:
Created https://github.com/celestiaorg/celestia-core/issues/900